### DISCUSSION TITLE
LTSP server load balancing / high performance

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#533] [Q&A] [answered]
I was looking to setup a LTSP server cluster (only 2 servers as of now) for high performance and load balancing. I would highly appreciate recommendations and directions in this regard.

Thanks

**Comment by alkisg** [comment, marked as answer]
The most difficult issue with load balancing is how to handle /home. In some cases a clustered file system would be more suitable (ceph, rbd, glusterfs); in others, an rsync-like approach (lsyncd, sending zfs snapshots over the network...). The other parts (DHCP, rootfs HA and balancing, user account management) are easier. Analyzing the setup and proposing the best solution might take a few hours, and implementing it might need a few days. Anyone interested in professional help on this may contact me, but unfortunately I do not have enough free time to personally document all the available solutions.

Of course others may have time to recommend/document some options; let's see...
